docs/rfcs/035-safekeeper-dynamic-membership-change.md

```diff
@@ -324,14 +324,20 @@ table! {
     timelines (tenant_id, timeline_id) {
         timeline_id -> Varchar,
         tenant_id -> Varchar,
+        start_lsn -> pg_lsn,
```

Comment:
how will the `start_lsn` be determined in the case of a timeline creation via import from postgres?

and in addition to the `start_lsn`, we also have to specify the postgres version upon timeline creation, so we'll probably have to save that in the table as well.

Reply:
> how will the start_lsn be determined in the case of a timeline creation via import from postgres?

Similarly, initial import to pageserver would give it.

for pg version: https://github.com/neondatabase/neon/pull/10514/commits/1b48724b613d05be8fe27ec5d7e0bcb6c1cd3f60

Reply:
> Similarly, initial import to pageserver would give it.

Yeah, agree. It will be retrievable via timeline detail endpoint once the import is finished.

@VladLazar have you made progress on speccing out how storage controller will be made fast-import-aware? (That _should_ be part of the Q1 work items to make it GA-ready)

Reply:
Haven't started. Wasn't aware of the requirement. I'll chase